ds3-cli-api/src/main/java/com/spectralogic/ds3cli/api/Arguments.java

```diff
@@ -13,28 +13,26 @@
  * ***************************************************************************
  */
 
-package com.spectralogic.ds3cli;
+package com.spectralogic.ds3cli.api;
 
 import ch.qos.logback.classic.Level;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
-import com.spectralogic.ds3cli.exceptions.BadArgumentException;
-import com.spectralogic.ds3cli.util.MetadataUtils;
+import com.spectralogic.ds3cli.api.exceptions.BadArgumentException;
+import com.spectralogic.ds3cli.utils.MetadataUtils;
 import com.spectralogic.ds3client.models.Priority;
 import com.spectralogic.ds3client.models.WriteOptimization;
 import org.apache.commons.cli.*;
 
 import java.util.ArrayList;
 import java.util.List;
 
-import static com.spectralogic.ds3cli.ArgumentFactory.*;
-
 public class Arguments {
 
     // Root args are added before first parse. All are optional
     private final static ImmutableList<Option> rootArgs
-            = ImmutableList.of(ENDPOINT, ACCESS_KEY, SECRET_KEY, COMMAND, PROXY, HTTP, INSECURE, PRINT_HELP, COMMAND_HELP,
-            VERBOSE, DEBUG, TRACE, LOG_VERBOSE, LOG_DEBUG, LOG_TRACE, PRINT_VERSION, VIEW_TYPE, RETRIES, BUFFER_SIZE);
+            = ImmutableList.of(ArgumentFactory.ENDPOINT, ArgumentFactory.ACCESS_KEY, ArgumentFactory.SECRET_KEY, ArgumentFactory.COMMAND, ArgumentFactory.PROXY, ArgumentFactory.HTTP, ArgumentFactory.INSECURE, ArgumentFactory.PRINT_HELP, ArgumentFactory.COMMAND_HELP,
+            ArgumentFactory.VERBOSE, ArgumentFactory.DEBUG, ArgumentFactory.TRACE, ArgumentFactory.LOG_VERBOSE, ArgumentFactory.LOG_DEBUG, ArgumentFactory.LOG_TRACE, ArgumentFactory.PRINT_VERSION, ArgumentFactory.VIEW_TYPE, ArgumentFactory.RETRIES, ArgumentFactory.BUFFER_SIZE);
 
```

Comment:
Out of curiosity, why is a "root argument"?  An argument you must provide?  Or an argument that is defined as separate from one you could use in an extension command?